api/flight_api.py

```diff
@@ -54,8 +54,8 @@ def flight_cheapest_date(origin: str, destination: str, response_access_token: s
 
 def cheapest_tickets(origin_country_code: str, destination_country_code: str, depart_date: str, arrival_date: str):
     cheapest_tickets_url = "https://travelpayouts-travelpayouts-flight-data-v1.p.rapidapi.com/v1/prices/cheap"
-    data_code = {"origin": "SIN", "page": "None",
-                 "currency": "USD", "destination": "ICN"}
+    data_code = {"origin": origin_country_code, "page": "None",
```

Comment:
Rename `data_code` to something more descriptive. Maybe `query_string` or `query_params`